app/worker/tasks/test__init__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from django.core import serializers
+from django.test import TestCase
+
+from app.models.donation import Donation
+from app.models.donor import Donor
+from app.worker import tasks
+
+
+class InitTestCase(TestCase):
+    def setUp(self) -> None:
+
+        return super().setUp()
+
+    def test_receiptor(self) -> None:
```

Comment:
Can we add one more test case here to check if we receipt more than one (2+) donations we get a zip file instead?

Reply:
Added in https://github.com/CodeTheChangeUBC/reBOOT/pull/343/commits/e3e75ee1516fb0f68caa4f57d62bb7040bdb72e9.